helpers/database.go

```diff
@@ -104,6 +104,7 @@ func DefineRandomFunction(ccdb *sql.DB, ctx context.Context) {
 
 func CleanupTestData(ccdb, uaadb *sql.DB, ctx context.Context, testConfig Config) {
 	deleteStatementsPostgres := []string{
+		//"DROP TABLE event_typess",
```

Comment:
Should be commented in

Reply:
This is not working at this place, working one is in line 176